Chapters/1-3-Chapter3.md

```diff
@@ -0,0 +1,259 @@
+## Chapter 3
+
+"Ara? Why would Melna need to know about that?"
+
+"The observers on Falni Peak are doing some study about the stellar surface and they said they've found something big and need to tell the senate right away"
+
+_She's keeping a secret_
+
+"Well, if she's not here, I should go back to call the capital."
+
+"Wait. Let me go with you."
+
+"Oh that's really not necessary -"
+
+"I need to know more; I won't slow you down" I plead to him while I ran off to fetch a new pair of trousers as quickly as I could.
+
+The late morning light was just enough for me to see while I dashed to the stable to get Laen-Laen, our Saffa. He was small for a Saffa, but at twelve times my own weight, he was more than enough to fit Jathi and me.
+
+"Laen-Laen, good morning!"
+
+He gave out a whoop as he woke up from his half-nap.
+
+"You're still asleep, Laen?"
+
+A loud "Ooo" sound filled the room as he stood up and yawned in my face.
+
+"Hey grass-breath, can you take me to Heiko central?"
+
+He snorted his disapproval with his fuzzy ears turned backward.
+
+"We can stop by Gilli's and get a cake."
+
+"Woop woop woop!" He said as he perked up and twisted his head left and right.
+
+"There we go. Now lets get you ready."
+
+I took a soft linen blanket from the wall and paused a moment to remember when Jathi and I used to walk with a younger Laen along the foggy coastline in the mornings. The three of us would play hide and seek on a wide open beach and just get lost shouting to each other in the gray abyss until the fog lifted.
+
+_Not the time for memories_. I snapped out and hopped on while rushing to the front gate where Jathi was waiting for me.
```

Comment:
```suggestion
But this is not the time for memories, so I snapped out and hopped on while rushing to the front gate where Jathi was waiting for me.
```

Will read more smoothly if you kept them in the same form. Since Liffa is the narrator here, some of hae thoughts can just be included in the narration instead of _thought tags._